main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"log"
 	"net"
 
+	"github.com/agarwalarjun123/NetNetra/packet"
```

Comment:
I don't understand the reason for doing this.

Reply:
I could not find a better way of issuing relative imports in forks in golang? We can anyway solve it when merging the changes into master

Reply:
But `packet` is surely not the right package name.
Since we will have flows as well. A more generic name would serve the purpose better.
And this was kinda early for a restructure PR. Since we were not even sure of the packet data to be indexed. As you can see I have proposed some changes already [here](https://github.com/darshkpatel/NetNetra/pull/13)
Let's hold on this PR for some time until we have everything finalized.

Reply:
BTW we should definitel use the ES wrapper you used :tada: